src/test/java/org/assertj/core/extention/SoftlyExtensionInstanceTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.extention;
+
+import org.assertj.core.annotations.Softly;
+import org.assertj.core.api.SoftAssertions;
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.assertj.core.extention.SoftlyExtension;
+import org.assertj.core.error.ShouldNotContainNull;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.util.AssertionsUtil.assertThatAssertionErrorIsThrownBy;
+
+@ExtendWith(SoftlyExtension.class)
+@DisplayName("SoftlyExtension hasInstance")
+class SoftlyExtensionInstanceTest {
+
+  private SoftAssertions softlyNotAnnotated1;
+  private SoftAssertions softlyNotAnnotated2;
+
+  @Softly
+  private SoftAssertions softlyAnnotated1;
+
+  @Softly
+  private SoftAssertions softlyAnnotated2;
```

Comment:
I don't see a use case where more than one SoftAssertions is needed.

Reply:
Me too, I just put it there because I thought someone might need it. But I can revert to an earlier implementation where I threw an exception when more that one field had the `@Softly` annotation.

Reply:
I agree, let's remove one of the annotated field and rename the other as `softly`